xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,114 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+SELECT_FIRST_UNWATCHED_ITEM CGUIWindowVideoNav::SelectFirstUnwatchedItem()
```

Comment:
You're just doing it to annoy me now aren't you? ;) lol

I did look to see if there were any existing enums prefixed with e and there are so I thought it would be ok.

But I have removed the prefix and changed the methods to GetSetting... as suggested